lib/archethic_web/controllers/api/transaction_controller.ex

```diff
@@ -25,21 +25,22 @@ defmodule ArchethicWeb.API.TransactionController do
           |> TransactionPayload.to_map()
           |> Transaction.cast()
 
-        case Archethic.send_new_transaction(tx) do
-          :ok ->
-            TransactionSubscriber.register(tx.address, System.monotonic_time())
+        case Archethic.search_transaction(tx.address) do
```

Comment:
By using search transaction we might fetch an entire transaction if exists. 
We should do a lightweight verification like using GetTransactionSummary message (acting like a header)

Reply:
Okay makes sense, for this then we should query storage nodes for the transaction summary. 
We can skip the quorom read if we are focusing on lightweight fast responses.

Reply:
I don't think we still need the quorum read, as we don't to avoid stale or inconsistent data.
The lightweight would be to reduce the payload to return.